figures/discovery/discovery-polling-interval.mmd

```diff
@@ -0,0 +1,10 @@
+stateDiagram-v2
+    state "Discovery mode determined" as discov
+    state "Developer pre-configures polling interval" as dev
+    state "Successful connection from Monolith to Balancer" as connection
+
+    [*] --> discov: Balancer initiates connection to Monolith
```

Comment:
The connection can't happen before discovery is performed.

You shouldn't be diagramming connections being made -- its not relevant to the issue. The polling interval determines how often discovery is polled for changes in available monoliths. This happens completely separate from the connection manager.